packages/create-snap-cli/package.json

```diff
@@ -0,0 +1,105 @@
+{
+  "name": "@metamask/create-snap",
```

Comment:
The folder name should be the same as the package name, so it should be `packages/create-snap`.